legacy/src/test/java/org/opensearch/sql/legacy/antlr/semantic/scope/EnvironmentTest.java

```diff
@@ -116,7 +119,7 @@ public void defineFieldSymbolShouldBeAbleToResolveAll() {
         Map<String, Type> typeByName = environment().resolveAll(Namespace.FIELD_NAME);
         assertThat(
             typeByName,
-            allOf(
+            Matchers.<Map<String, Type>>allOf(
```

Comment:
Same as `SemanticAnalyzerBasicTest.java` this file is different in `upstream/main`.

Reply:
Thank you, fixed in 65ec516.